scripts/cleanup-white-noise.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+sed -i 's/[ \t]*$//' "$@"
```

Comment:
Nice! I think we might want to have it in `Makefile` as well. We can hook it on the CI in later PR.

Reply:
format `*.go`?